### ISSUE TITLE
cmake_external: failed to copy file if filename contains space.

### ISSUE THREAD

**Issue by ukai** [bazelbuild/rules_foreign_cc#406]
I found the following error in CMake.log

```
Can not copy /src/workspace/.cache/bazel/_bazel_ukai/a08c2e4811c846650b733c6fc815a920/sandbox/processwrapper-sandbox/1/execroot/test/external/cmake_linux/cmake-3.16.2-Linux-x86_64/doc/cmake/html/_sources/generator/Eclipse
Can not copy CDT4.rst.txt
Can not copy /src/workspace/.cache/bazel/_bazel_ukai/a08c2e4811c846650b733c6fc815a920/sandbox/processwrapper-sandbox/1/execroot/test/external/cmake_linux/cmake-3.16.2-Linux-x86_64/doc/cmake/html/_sources/generator/NMake
Can not copy Makefiles
Can not copy JOM.rst.txt
```

tools/build_defs/shell_toolchain/toolchains/impl/linux_commands.bzl uses
```
  local children=$(find -H "$1" -maxdepth 1 -mindepth 1)
  for child in $children; do
```

but it should be something like
```
 SAVEIFS="$IFS"
 IFS=$'\n'
 local children=($(find -H "$1" -maxdepth 1 -mindepth 1))
 IFS="$SAVEIFS"
 for child in "${children[@]}"; do
```

also, split_argument in tools/build_defs/shell_script_helper.bzl drops quote, so
 cp "$1" "$2"
becomes
 cp $1 $2
in actual script.

```
 parts.append(quoted)
```
should be
```
 parts.append("\"" + quoted + "\"")
```

?

**Comment by attilaolah** [comment]
I just hit my first repo that contains filenamse whit spaces 😢 

Would be awesome to fix this!

**Comment by kcherneha** [comment]
Have the same issue with cmake_external for libblepp library.
What is the current status of this issue? When it will be fixed on master branch?

**Comment by attilaolah** [comment]
I think when #416 is merged this should be fixed.

**Comment by attilaolah** [comment]
#416 was just merged in in 78dd474, this should now be fixed.
